src/DatatypeChannels.ASB/Consumer.fs

```diff
@@ -114,8 +122,7 @@ let mkNew (options: Options)
                 | Some (receiver, _, msgCtxs) ->
                     for KeyValue(_,msgCtx) in msgCtxs.ToArray() do
                         msgCtx.Close()
-                    msgCtxs.Clear()
```

Comment:
There used to be a purpose to clearing the collection, but since I rewrote the locks upkeep there's no need.